modules/nf-core/software/bwamem2_index.nf

```diff
@@ -3,18 +3,18 @@ include { initOptions; saveFiles; getSoftwareName } from './functions'
 params.options = [:]
 def options    = initOptions(params.options)
 
-environment = params.enable_conda ? "bioconda::bwa-mem2=2.0" : null
-container = "quay.io/biocontainers/bwa-mem2:2.0--he513fc3_1"
-if (workflow.containerEngine == 'singularity' && !params.pull_docker_container) container = "https://depot.galaxyproject.org/singularity/bwa-mem2:2.0--he513fc3_1"
-
 process BWAMEM2_INDEX {
     tag "${fasta}"
 
     publishDir params.outdir, mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:"false") }
 
-    conda environment
-    container container
+    conda (params.enable_conda ? "bioconda::bwa-mem2=2.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bwa-mem2:2.0--he513fc3_1"
```

Comment:
The tool version here and in the actual mapping module is not the same. should be 2.1 as well